.github/workflows/helm.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Helm Tests
+on:
+  push:
+    paths:
+      - 'deploy/helm/**'
+  pull_request:
+    paths:
+      - 'deploy/helm/**'
+  # Allows you to run this workflow manually
+  workflow_dispatch:
+jobs:
+  check-for-doc-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
+      - name: Regenerate Helm Docs
+        run: |
+          docker run --rm \
+             -v "$(pwd):/helm-docs" \
+             -u "$(id -u)" \
+             jnorwood/helm-docs:latest # This should probably be a specific version
```

Comment:
I initially used `make helm` but the action failed due to `make` not being installed. I'll follow up and see how to do that so that we share code between the Makefile and worfkflow.